### PR TITLE
adding note for password change and re-encryption

### PR DESCRIPTION
**Pull request by StanoRiga**
source: https://community.cryptomator.org/t/what-happens-exactly-when-i-change-password-of-a-vault

### CODE REVIEW

File: docs/desktop/vault-management.md [hunk 1]
```diff
@@ -27,3 +27,13 @@ You can now delete the folder that contains the `masterkey.cryptomator` file as
...
 ![How to reorder vaults](../img/move-vaults.gif)
+
+# Change Vault Password
+
+The password can be changed by right-clicking on a vault while it is locked. Currently, it’s not possible to change the password while the vault is unlocked.
+
+![How to change vault password](../img/change-password-desktop.png)
+
+!!! note
+    The password is used to derive a [KEK 97](https://en.wikipedia.org/wiki/Glossary_of_cryptographic_keys){: rel="external"}, which is then used to encrypt futher keys. The KEK changes, but the keys encrypted with the KEK will stay the same. The actual files will not get re-encrypted, meaning you can not upgrade a weak passphrase to a stronger one once the data has been synced to a service that allows recovery of older versions of the masterkey file.
```
Comment (overheadhunter) [inline docs/desktop/vault-management.md]: Why the "97"?

Comment (StanoRiga) [reply docs/desktop/vault-management.md]: Ah damn it. copy/paste error. 
(97 is the amount of click your link has in the forum thread).
I'm sorry.
Shall I fixe it and start a new pull request, or can you just delete the " 97"?

Comment (overheadhunter) [reply docs/desktop/vault-management.md]: You can add a new commit, it will be ammended to this PR automatically

**Comment by overheadhunter** [comment]
Thank you!

**Pull request merged**